exercises/practice/pythagorean-triplet/.meta/example.rb

```diff
@@ -1,52 +1,21 @@
-class Triplets
-  attr_reader :factors, :sum
-  def initialize(conditions)
-    min = conditions.fetch(:min_factor) { 1 }
-    max = conditions.fetch(:max_factor)
-    @sum = conditions[:sum]
-    @factors = (min..max).to_a
-  end
-
-  def to_a
+class PythagoreanTriplet
+  def self.triplets_with_sum(sum)
     triplets = []
-    each_triplet do |triplet|
-      triplets << triplet if select?(triplet)
+    (2..(sum/2)).to_a.reverse.each do |c|
+      a = 1
+      b = sum-c-a
```

Comment:
```suggestion
      b = sum - c - a
```

Space around operators, this looks different than other areas of this same file.

Reply:
I must have forgotten to run rubocop autocorrect on it.